### ISSUE TITLE
Unable to use the library independent of any frameworks.

### ISSUE THREAD

**Issue by gnanakeethan** [Shopify/shopify-api-js#906]
## Overview

I am at the moment attempting to use this library for Admin API with svelte. 

It appears that there is a tight integration between node typing and the values. 

In comparison, the Python library offered has some methods which allow basic functions to operate independent. 

Is there any chance I can use Admin API without using a full backend "express" etc. 

Thanks

**Comment by rodrigogsqquid** [comment]
I'd like to add up to this request. The current state of the library makes it very hard for nodejs developers to use it outside of major frameworks such as express or koa. 

In my case I need to use it in an AWS Lambda context and I'm definitely far from adding the overhead of any such frameworks. 

The way the current library is designed seem to be very tied to the paradigm express has.

Is writing an adapter the go-to solution for this case? 
If so, would you be so kind to share with us a bit more context on what's expected from it?

Thanks a lot

**Comment by oneEyedSunday** [comment]
> Yeah, im migrating from version 5 to version 7 (because of the Api deprecations) and it feels so overwhelming

FYI just follow the migration guide.

TLDR

```javascript

require('@shopify/shopify-api/adapters/node');

```
